File: bialydunajec-registrations/application/src/main/kotlin/org/bialydunajec/registrations/application/dto/BankTransferDetailsDto.kt
package org.bialydunajec.registrations.application.dto

import org.bialydunajec.ddd.domain.base.validation.constraints.NullOrNotBlank


data class BankTransferDetailsDto(
        @field:NullOrNotBlank
        val accountNumber: String?,

        @field:NullOrNotBlank
        val accountOwner: String?,

        @field:NullOrNotBlank
        val accountOwnerAddress: String?,

        @field:NullOrNotBlank
        val transferTitleTemplate: String?
)

File: frontend-adminpanel/projects/bialydunajec-admin/src/app/camp-registrations/service/rest/dto/bank-transfer-details.dto.ts
export class BankTransferDetailsDto {
  accountNumber?: string;
  accountOwner?: string;
  accountOwnerAddress?: string;
  transferTitleTemplate: string;


  constructor(accountNumber: string = null, accountOwner: string = null, accountOwnerAddress: string = null, transferTitleTemplate: string = null) {
    this.accountNumber = accountNumber;
    this.accountOwner = accountOwner;
    this.accountOwnerAddress = accountOwnerAddress;
    this.transferTitleTemplate = transferTitleTemplate;
  }
}
